ios/RunnerTests/RunnerTests.swift

```diff
@@ -0,0 +1,12 @@
+import Flutter
+import UIKit
+import XCTest
+
+class RunnerTests: XCTestCase {
+
+  func testExample() {
+    // If you add code to the Runner application, consider adding tests here.
+    // See https://developer.apple.com/documentation/xctest for more information about using XCTest.
```

Comment:
Are we using this test?

Reply:
The RunnerTests class is a template for unit tests. This file allows to write tests for native iOS components. Though I am not testing any native code, so don't need it. If I have to test iOS components that will be created with flutter code it will be tested on **test/** folder in main directory.